app/src/main/res/values/strings.xml

```diff
@@ -40,17 +40,17 @@
     <string name="allow">Allow</string>
     <string name="clipboard_cleared">Clipboard cleared</string>
     <string name="clipboard_error_title">Clipboard error</string>
-    <string name="clipboard_error">Some Samsung Android phones have a bug in the clipboard implementation causing copying from apps to fail. More details:</string>
-    <string name="clipboard_error_clear">Clearing the clipboard failed</string>
+    <string name="clipboard_error">Some Samsung Android phones crash apps when using clipboard. More details:</string>
```

Comment:
What ? No, there is no crash, there is not even an exception. 
But the samsung overlay handles the clipboard differently and we can not delete clipboard elements in a conventional way.